assets/docs/rishit-dagli/plant-disease/default/lite/1.md

```diff
@@ -0,0 +1,29 @@
+# Lite rishit-dagli/plant-disease/default/tfjs/1
+TF Lite deployment of rishit-dagli/plant-disease/1 .
+
+<!-- parent-model: rishit-dagli/plant-disease/1 -->
+<!-- asset-path: https://storage.googleapis.com/rishit-dagli/plant-disease-model/model.tflite -->
+
+[![Open Colab notebook]](https://colab.research.google.com/github/Rishit-dagli/Greenathon-Plant-AI/blob/main/notebooks/plant_disease_TFLite.ipynb)
```

Comment:
Could you also remove this colab link, and the TFJS one? They don't add value to help others use these models.

Reply:
Thanks for pointing this out, this makes sense!